### DISCUSSION TITLE
ce-v1.9.35

### DISCUSSION THREAD

**Discussion by AppsmithCN** [cloudtogo/pageplug#79] [Show and tell]
- 功能变化
1、MSSQL数据库支持Schemas模式
2、新的交互界面和管理员注册交互界面
3、支持在应用编辑状态下，复制应用到新的应用组
4、添加逻辑?embed=true&navbar=true
5、文件上传组件对xls、json、tsv文件类型的支持
6、支持在应用程序内对用户进行邀请和分配角色
7、添加对yaml工作流程文件、gitconnect验证的支持
8、向 git disrad 流程添加reabase选项
9、增加了api运行时加载动画的显示
10、配置nginx在启动时配置CDN，首次加载的速度变得更快了
11、查询功能迭代，默认从数据库中的第一个表进行查询
![image](https://github.com/cloudtogo/pageplug/assets/104495020/0df51f2b-3136-4426-89b8-c277ed8f763f)
12、Git支持显示对应用主题和设置修改的显示
13、将图标代码合并为remix-icons、blueprint-icons和svg-icons三块内容，构建时间变得更快了
![image](https://github.com/cloudtogo/pageplug/assets/104495020/454cb9d6-0829-44f1-b4ab-43d8badc3b5e)
14、优化了Linting性能，响应时间从2秒缩短到1.2秒
15、支持setter的方式来更新组件的属性(例如input组件，支持setVisibility设置组件的可见性、支持setDisabled设置组件的禁用状态、支持setValue动态设置组件的值、支持setRequired是否设置组件为必填)
![image](https://github.com/cloudtogo/pageplug/assets/104495020/f8a5d2f3-a225-40da-936f-ccf17a053db6)
16、增加了APPSMITH_SERVER_TIMEOUT字段，支持自定义nginx超时值
17、setter方法也支持linting错误的显示
18、新增达梦、TIDB数据库
![image](https://github.com/cloudtogo/pageplug/assets/104495020/465aa793-02f2-4fe1-a6c7-43d51003a1cc)
19、sql智能提醒支持开启及关闭
![image](https://github.com/cloudtogo/pageplug/assets/104495020/731a05c7-c283-4fff-bdc8-612819252596)
20、组件输入框支持浮动样式，编写代码体验更好了
![image](https://github.com/cloudtogo/pageplug/assets/104495020/a5811102-911f-43b4-9689-dfcc77259e42)
21、导航样式支持更多的内容，顶部、侧边、行内、堆叠的效果，也支持配置logo图标
![image](https://github.com/cloudtogo/pageplug/assets/104495020/f1478037-5fb2-4bb5-bec5-151891c6b717)
22、画布新增响应式布局，页面的布局设计
![image](https://github.com/cloudtogo/pageplug/assets/104495020/c6091294-67b4-48e6-bceb-2d891f3e18df)
23、多行查询返回结果改造，执行多条sql后，支持只直找到第一个结果集
24、新增微信单点登陆功能

- Bug修复
1、修复了无效数据传递在类别滑块小部件报错的问题
2、修复了邮件不支持.info、.tech的验证问题
3、优化了DSL调用比其他API调用慢的问题
4、修复了使用curl命令创建api时，key和value值不可填的问题
5、修复了部分特定场景使用了不正确的workspaceid问题
6、屏蔽了应用发布后应用程序的接口调用失败等消息提示
7、修复了树形单选和树形多选中children字段的智能提示
8、修复了Mongock进程中的写入错误而无法启动DocumentDB的问题
9、修复了允许复制标签页中的标签所带来的报错错误
10、修复了查询时表格高度的问题
11、在datasourceStorageStructure集合上放置了更具体的唯一索引
12、允许localhost作为图片组件的url
13、修复了删除Jsobkect时抛出错误“无法执行函数”的问题
14、优化了应用系统图标一致性的问题
15、修复了表格组件搜索框无法搜索url的问题
16、添加了JSobject中对括号自动完成的支持
17、修复了Mysql过时连接的报错问题
18、增加选项卡的宽度支持更多的文本（从138px增加到180px）
19、修复了日志时间不一致的问题
20、修复了Curl导入时，URL中出现大括号错误的问题
21、修复了组件的文本不支持复制的问题

更多信息查阅PP文档https://docs.pageplug.cn/%E4%BB%8B%E7%BB%8D/%E7%89%88%E6%9C%AC%E6%97%A5%E5%BF%97

<hr /><em>This discussion was created from the release <a href='https://github.com/cloudtogo/pageplug/releases/tag/'>ce-v1.9.35</a>.</em>

**Comment by hubsvn277** [comment]
can translate to english ? current appsmith using english but Pageplug is chinese and it not friendly with me but i want to use pageplug
